rcl/src/rcl/logging.c

```diff
@@ -76,6 +77,12 @@ rcl_logging_configure_with_output_handler(
   if (log_levels) {
     default_level = (int)log_levels->default_logger_level;
     rcutils_logging_set_default_logger_level(default_level);
+    if (RCUTILS_LOG_SEVERITY_UNSET != default_level) {
+      rmw_ret_t rmw_status = rmw_set_log_severity((rmw_log_severity_t)default_level);
+      if (RMW_RET_OK != rmw_status) {
+        return RCL_RET_ERROR;
```

Comment:
if implementation does not support `rmw_set_log_severity`, this will generate exception `RCLError` to user application to fail initialization. I think it is up to implementation to support `rmw_set_log_severity` or not. so how about print error to show `it cannot enable rmw implementation logger` instead of returning error?

Reply:
Good point, done in e8a34876bde0621555ba8a4b73836c1ce10e2b35